src/config/nunjucks.ts

```diff
@@ -0,0 +1,81 @@
+import addLanguageParam from "@govuk-one-login/frontend-language-toggle";
+import { Application } from "express";
+import i18next from "i18next";
+import nunjucks, { Environment } from "nunjucks";
+import { kebabCaseToPascalCase } from "../app/shared/stringHelper";
+import config from "../lib/config";
+
+interface FilterContext {
```

Comment:
I can't actually work out where this comes from, and whether there's a better type definition for it in one of the libraries... Open to suggestions!